lib/iris/tests/integration/test_pp.py

```diff
@@ -711,5 +711,27 @@ def test_singular_longitude(self):
         self.assertTrue(cube.coord('longitude').has_bounds())
 
 
+@tests.skip_data
+class TestLoadPartialMask(tests.IrisTest):
+    def test_data(self):
+        # Ensure that fields merge correctly where one has a mask and one
+        # doesn't
```

Comment:
Perhaps we could have a full stop at the end of this second comment line. At the moment it looks like you forgot to finish the comment!